docker-compose/run_selenium_tests_remote.sh

```diff
@@ -50,7 +50,8 @@ echo_msg
 # Set bash builtins for safety
 set -e -u -o pipefail
 
-export USE_NEW_PERM_SCREEN=false
+USE_LOGIN_WITH_MEDICARE_BUTTON="${USE_LOGIN_WITH_MEDICARE_BUTTON:-}"
```

Comment:
I can't seem to get a run of `docker-compose/run_selenium_tests_remote.sh TEST` to ever record `USE_LOGIN_WITH_MEDICARE_BUTTON=false`, even when I set `USE_LOGIN_WITH_MEDICARE_BUTTON='false'` prior to running. This might not be working correctly, or maybe I'm not setting the variable correctly. Can you check that out, and let me know if there is a way to force `USE_LOGIN_WITH_MEDICARE_BUTTON='false'` against TEST?

Reply:
I tried this, I added `USE_LOGIN_WITH_MEDICARE_BUTTON=false` to the env file, but in the output I still see `USE_LOGIN_WITH_MEDICARE_BUTTON= true`.